doc/coding-conventions.xml

```diff
@@ -254,10 +254,13 @@ bound to the variable name <varname>e2fsprogs</varname> in
   dash) — e.g., <literal>"hello-0.3.1rc2"</literal>.</para></listitem>
 
   <listitem><para>If a package is not a release but a commit from a repository, then
-  the version part of the name <emphasis>must</emphasis> be the date of that 
+  the version part of the name should be the date of that 
   (fetched) commit. The date must be in <literal>"YYYY-MM-DD"</literal> format.
-  Also append <literal>"unstable"</literal> to the name - e.g.,
-  <literal>"pkgname-unstable-2014-09-23"</literal>.</para></listitem>
+  Also append <literal>"unstable"</literal> to the name — e.g.,
+  <literal>"pkgname-unstable-2014-09-23"</literal>.
+  If the package is a pre-release snapshot, the version can reflect this instead
```

Comment:
This can be applied only when the maintainers of the project clearly communicate the next version number and maybe I should clarify this but I didn't want to be too verbose. What do you think? Should I add the clarification?

In the use-case I have in mind (Coq), there clearly is a clear communication on the next version with a roadmap...

But in all the other cases, I'm very fine for continuing to apply the previous policy.